recipes/modin/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "modin" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: modin
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cb11d4f62453d47f0d014e2fd9b90fd001c6738ae377e929ae1996d860767c30 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [py<35]
```

Comment:
I would suggest removing the `skip` line and make this package `noarch: python`, and then change `python` to `python >=3.5` under `host` and `run`